docs/protocols.md

```diff
@@ -98,6 +98,16 @@ On some configurations only one JNLP3 slave per IP address can be connected.
 * [JENKINS-34121](https://issues.jenkins-ci.org/browse/JENKINS-34121) -
 JNLP3 cannot be used on IBM Java, which doesn't support AES/CTR/PKCS5Padding.
 
+## Plugin protocols
+
+### Remoting Kafka Plugin
+
+* [Remoting Kafka Plugin](https://github.com/jenkinsci/remoting-kafka-plugin) uses Kafka as fault-tolerant communication layer to support command invocation between Jenkins master and agent.
+* The plugin gets rid of current direct TCP connection between master and agent.
+* Below picture is the overview architecture of the plugin.
```

Comment:
I would rather add `More info` link and reference a documentation section in the Remoting Kafka plugin

Reply:
@oleg-nenashev Should I create a new markdown file for the plugin documentation or should I reference the `README`?

Reply:
New page could be preferable